src/app/components/home/home-top-banner.component.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+
+.home-header-container {
+  background: linear-gradient($color-cerise-600, $color-primary-cerise);
+  color: $white;
+  display: flex;
+  flex-flow: row-reverse;
+
+  @media screen and (max-width: $md-device-width) {
+    flex-flow: column;
+  }
+
+  .home-header-map-image {
+    padding-right: 0;
```

Comment:
Properties should be ordered width, padding-right

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)